trapdata/antenna/registration.py

```
@@ -13,30 +15,49 @@
 from trapdata.settings import Settings, read_settings
 
 
+def _get_version() -> str:
+    """Return the ami-data-companion package version, or 'unknown'."""
+    try:
+        from importlib.metadata import version
+
+        return version("trapdata")
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)